runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -236,15 +336,15 @@ public LayoutWorkflowManager removeLogunitServer(String endpoint)
      * @return Workflow manager
      * @throws LayoutModificationException Cannot remove non-replicated logunit
      */
-    public LayoutWorkflowManager removeLogunitServers(Set<String> endpoints)
+    public LayoutBuilder removeLogunitServers(Set<String> endpoints)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)